src/confluent_kafka/src/Admin.c

```diff
@@ -3624,9 +3628,13 @@ static PyObject *Admin_c_MemberDescription_to_py(const rd_kafka_MemberDescriptio
                              "client_id",
                              rd_kafka_MemberDescription_client_id(c_member));
 
-        cfl_PyDict_SetString(kwargs,
-                             "host",
-                             rd_kafka_MemberDescription_host(c_member));
+        if (rd_kafka_MemberDescription_host(c_member)) {
+                cfl_PyDict_SetString(kwargs,
+                                "host",
+                                rd_kafka_MemberDescription_host(c_member));
+        } else {
+                cfl_PyObject_SetItemString(kwargs, "host", Py_None);
+        }
```

Comment:
Please revert this change:

1. This change is not related to describe_topic
2. We can't reproduce this
3. It shouldn't be null as per the protocol

Reply:
+1 : In this case it's the member host and is not nullable by the protocol

https://github.com/apache/kafka/blob/18f2589c1e087654c0cdd46d2473246f37f0b66a/clients/src/main/resources/common/message/DescribeGroupsResponse.json#L57